pldmd/pldmd.cpp

```diff
@@ -452,6 +455,9 @@ int main(int argc, char** argv)
 
     bus.attach_event(event.get(), SD_EVENT_PRIORITY_NORMAL);
     bus.request_name("xyz.openbmc_project.PLDM");
+
+    pldm::deserialize::restoreDbusObj(hostPDRHandler);
+
```

Comment:
@lxwinspur we should restore objects only when host is up. we should not restore any dbus objects when host is not up.

Reply:
we can use isHostUp() function - but make sure we execute this only after we get a reply from getPLDMVersion command.

Reply:
> we can use isHostUp() function - but make sure we execute this only after we get a reply from getPLDMVersion command.

Done